src/main/controllers/StoredResponseAppConfirmationController.ts

```diff
@@ -0,0 +1,33 @@
+import { Response } from 'express';
+
+import { AppRequest } from '../definitions/appRequest';
+import { ErrorPages, TranslationKeys } from '../definitions/constants';
+import { AnyRecord } from '../definitions/util-types';
+import { getLogger } from '../logger';
+
+import { getAppDetailsLink, getCancelLink } from './helpers/LinkHelpers';
+import { getLanguageParam } from './helpers/RouterHelpers';
+
+const logger = getLogger('StoredResponseAppConfirmationController');
+
+export default class StoredResponseAppConfirmationController {
+  public async get(req: AppRequest, res: Response): Promise<void> {
+    const languageParam = getLanguageParam(req.url);
+
+    const translations: AnyRecord = {
+      ...req.t(TranslationKeys.COMMON, { returnObjects: true }),
+      ...req.t(TranslationKeys.STORED_APPLICATION_CONFIRMATION, { returnObjects: true }),
+    };
+
+    if (req.params.appId === undefined) {
+      logger.error('Application ID not found');
+      return res.redirect(`${ErrorPages.NOT_FOUND}${languageParam}`);
+    }
+
+    res.render(TranslationKeys.STORED_APPLICATION_CONFIRMATION, {
```

Comment:
I'm not following how you can add this controller without a view. What page does it load?

Reply:
It is using the same view which was previously created:
https://github.com/hmcts/et-sya-frontend/blob/Rule-92/src/main/views/stored-application-confirmation.njk